include/libtrx/game/game_string.def

```diff
@@ -0,0 +1,3 @@
+GS_DEFINE(OSD_POS_GET, "Room: %d\nPosition: %.3f, %.3f, %.3f\nRotation: %.3f,%.3f,%.3f")
+GS_DEFINE(OSD_CURRENT_HEALTH_GET, "Current Lara's health: %d")
+GS_DEFINE(OSD_CURRENT_HEALTH_SET, "Lara's health set to %d")
```

Comment:
No. These provide reasonable defaults in case the player or level builder removes the strings from the gameflow. Our tools automatically backpropagate these as default overrides to the gameflow files to highlight what can be overridden, but they are not strictly necessary as they ship the same strings that we provide here in the executable itself.